docs/markdown/Reference-manual.md

```diff
@@ -1522,6 +1522,11 @@ page](Configuration.md) It has three methods:
   value has not been set returns `default_value` if it is defined
   *(added 0.38.0)* and errors out if not
 
+- `get_unquoted(varname, default_value)` returns the value of `varname`
+  but without surrounding double quotes (`"`). If the value has not been
+  set returns `default_value` if it is defined and errors out if not.
+  Available since 0.43.0
```

Comment:
It looks like this wasn't actually merged until after 0.43.0, and so first makes an appearance in 0.44.0